gclient/src/api/calls.rs

```diff
@@ -355,6 +360,22 @@ impl GearApi {
                 .await?;
         }
 
+        let dest_gas_total_issuance =
+            dest_node_api.0.api().total_issuance().await.or_else(|e| {
+                if let GsdkError::StorageNotFound = e {
+                    Ok(0)
+                } else {
+                    Err(e)
+                }
+            })?;
+
+        dest_node_api
+            .0
+            .set_total_issuance(
+                dest_gas_total_issuance.saturating_add(src_program_reserved_gas_total),
+            )
+            .await?;
```

Comment:
@breathx I do not like this piece of code as well as the above one. It is unsafe in term of atomicity as well as the above one where we update balances on the dest node. The base value which is used for later update can change between 2 calls. For the source node we have solved similar issues using some fixed block number, but I can't make up anything what we can do for the destionation node

Reply:
seems any "mutate" fns are impossible here so nothing to do :(